src/api/posts/get/getPosts.js

```diff
@@ -1,15 +1,25 @@
+//imports
 import axios, * as others from 'axios'
+import { ref } from 'vue'
 
+//data
```

Comment:
in API block you only make requests, if you need to import ref, import it from state binded with specific state